app/src/main.js

```diff
@@ -19,7 +26,22 @@ const sendUrlToRouter = (url) => {
   }
 };
 
+/**
+ * Sends an event to client application
+ * @param {String} locale - the 2 letter name of the local
+ */
+const sendDetectedLang = (locale) => {
```

Comment:
`sendDetectedLang` looks very similar to`sendUrlToRouter`. The common pattern could be abstracted into a function, e.g.: `sendEvent(eventName, data)`

Reply:
we don't need `event` in `sendDetectedLang` but we don't need it in `sendUrlToRouter`